src/components/Tables/TableFilterBar.tsx

```diff
@@ -30,7 +30,7 @@ const StyledFilterBar = styled.input`
     border: 1px solid ${(props) => props.theme.secondaryBackgroundColor};
     box-shadow: 0 0 1px ${(props) => props.theme.secondaryBackgroundColor};
     padding: 4px 4px 4px 8px;
-    border-radius: 0.8rem;
+    border-radius: 4rem;
```

Comment:
Just wondering if it worth replacing it with the mui input that we have created before for consistency

Reply:
I agree, and I had made an attempt to do this. It's a little tricky and requires a bit of finicking about, but I've put it on a to-do list for myself